pages/common/sass.md

```diff
@@ -1,6 +1,7 @@
 # sass
 
 > Converts SCSS or Sass files to CSS.
+> Homepage: <https://sass-lang.com/documentation/cli/dart-sass>.
```

Comment:
I'm not sure we'll want to do this (link to the CLI documentation vs. the project's homepage) on every case. I've added a comment in the main thread proposing a discussion about that.